src/main/java/core/gui/model/SpielerCBItem.java

```diff
@@ -51,16 +52,14 @@ public SpielerCBItem(String text, float poswert, Player player, boolean useCusto
     public final Component getListCellRendererComponent(JList jList, int index, boolean isSelected) {
         final Player player = getSpieler();
 
-        // Kann für Tempspieler < 0 sein && player.getSpielerID () > 0 )
         if (player != null) {
             m_clEntry.updateComponent(player, HOVerwaltung.instance().getModel().getLineupWithoutRatingRecalc()
                             .getPositionBySpielerId(player.getSpielerID()), getPositionsBewertung(), m_bAlternativePosition,
                     m_sText);
 
-            if (m_bMultiLine) {
-                m_clEntry.getComponent(isSelected).setPreferredSize(new Dimension(m_clEntry.getComponent(isSelected).getPreferredSize().width, PLAYER_COMBO_HEIGHT));
-            }
-            return m_clEntry.getComponent(isSelected);
+            JComponent comp = m_clEntry.getComponent(isSelected, index==-1);
```

Comment:
@weisJ I just saw that you did commit https://github.com/weisJ/darklaf/commit/a9e0ce3a220f4c56027d5f28d1fc70a0a022a3f6

does it mean you reverted to old behaviour and what I did in the PR will eventually not be necessery ?

Reply:
The change was made to make the `index == -1` check functional. The changes are still necessary.